sway-book/src/documentation/misc/workarounds/importing.md

```diff
@@ -0,0 +1,16 @@
+# Importing
+
+In [external libraries](../../language/program-types/libraries/external.md) we have looked at how a library can be imported into a project so that code can be reused.
+
+When it comes to importing only external libraries can be imported and any other type of program will result in an error.
+
+This means that the following projects cannot be imported:
+
+- [contracts](../../language/program-types/contract.md)
+- [internal libraries](../../language/program-types/libraries/internal.md)
```

Comment:
Idk how I feel about us saying you can't import internal libraries.  IMO it's a bit confusing because don't we import them using the `dep` keyword.